tests/utils.py

```diff
@@ -55,6 +56,29 @@ def convert_ts_to_int_timestamp(ts: TSDataset, shift=0):
     return ts
 
 
+def convert_ts_index_to_freq(ts: TSDataset, freq: str, shift=0):
```

Comment:
As I understand, it doesn't work with int timestamp?

Reply:
Yes, this is just to convert from one frequency to another, similar to `convert_ts_to_int_timestamp` for conversions to int timestamps.